package.json

```diff
@@ -112,6 +112,7 @@
     "zod-to-json-schema": "^3.23.3"
   },
   "peerDependencies": {
+    "@googleapis/customsearch": "*",
```

Comment:
Users should use the version that we developed our extension against. So you might use `^3.2.0` or `^3.0.0` (but test it first if you decide to go with 3.0.0)